NEWS

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+## Current
+### Features:
+### Bugfixes:
+
+## 2.7.0 (June 13, 2023)
+### Features:
+#### test
+ * Added gtest suite including port of existing tests
+#### kernel
+ * Added faulting of multiple pages at once, forward and backward
+ * Added compilation fixes for PPC on Ubuntu 18.04
+ * Added vma_iterator and provided implementation for kernel < 6.2
```

Comment:
Move to bugfixes as "Fixed vma_iterator compilation issue on kernel ..."

Reply:
fixed